src/back/TicketManager/src/Core/TicketManager.Core.Domain/Events/SeatReservation.cs

```diff
@@ -0,0 +1,4 @@
+namespace TicketManager.Core.Domain.Events;
+
+public record SeatReservation(Guid Id, Guid SectorReservationId, int ReservedSeatNumber, DateTime CreationDate)
```

Comment:
Actually we don't need `Id` and `SectorReservationId` here. We care only about value of this object and we don't care at all about its identity. Of course `Id` and `SectorResevationId` will be created in db schema by ef, but we don't have to see them.

Also we need `UserId` to know which user made reservation.